Changes to fix issue 737

LethalCorpse opened the pull request:
This should take care of it until you can rewrite the PP code.

midgetspy commented:
Hrm your line endings are messed up, I can't tell what the changes are... try reverting and committing again with windows line endings? (Git can take care of it for you with autocrlf)

wafflecopter committed:
save some real estate and add a little hint

wafflecopter committed:
add jump-to-season links

wafflecopter committed:
another br we seem ok without

wafflecopter committed:
add links back to top

wafflecopter committed:
Merge branch 'master' of git://github.com/midgetspy/Sick-Beard

* 'master' of git://github.com/midgetspy/Sick-Beard:
  Added some logic to handle multiple conflicting multi-episode results better

wafflecopter committed:
better hint verbage

midgetspy committed:
Tweaked header message spacing

wafflecopter committed:
make top link disappear when season header does

LethalCorpse committed:
hmm... messed up \n might have been due to google chrome. Trying firefox.

Fixed continuous overwrite of existing file and renaming of parent folder to "UNDERSIZED"
